### PR TITLE
Add instructions for installation in arch linux

### PR DESCRIPTION
**Pull request by ChaosKid42**
I created a AUR package description for arch linux. This PR adds the necessary documentation.

### CODE REVIEW

File: installation/linux.md [hunk 1]
```diff
@@ -198,6 +198,34 @@ Optionally, you may in addition install the legacy add-ons package `openhab2-add
 This package contains 1.x bindings, for which there is already a 2.x version available.
 This might be useful if you're [coming from openHAB 1.x]({{base}}/tutorials/migration.html) for example.
 
+#### Arch Linux
+
+{% include collapsible/body.html %}
+
+The Arch User Repository ([AUR](https://wiki.archlinux.org/index.php/Arch_User_Repository)) is a community-driven repository for Arch users. It contains package descriptions (PKGBUILDs) that allow you to compile a package from source with makepkg and then install it via pacman.
```
Comment (Confectrician) [inline installation/linux.md:205]:
```suggestion
The Arch User Repository ([AUR](https://wiki.archlinux.org/index.php/Arch_User_Repository)) is a community-driven repository for Arch users. 
It contains package descriptions (PKGBUILDs) that allow you to compile a package from source with makepkg and then install it via pacman.
```

One line per sentence please.

**Review by Confectrician — CHANGES_REQUESTED** [review]
Hey @ChaosKid42 thanks for your contribution. 👍 

I have one small comment, but then we are good to go. :)

File: installation/linux.md (hunk 1 above)
Comment (ChaosKid42) [reply installation/linux.md]: @Confectrician Sure. Done

**Review by Confectrician — APPROVED** [review]
Thanks. 👍

**Pull request merged**